### PR TITLE
MNT: Updates for level b files.

### PR DESCRIPTION
**Pull request by zssherman**

### CODE REVIEW

**Comment by zssherman** [comment]
@scollis When you get a chance.

File: cmac/default_config.py [hunk 1]
```diff
@@ -360,8 +360,8 @@
     # CACTI C-SAPR 2 field names.
     'cacti_csapr2_ppi': {
         # Radar field names
-        'input_zdr': 'differential_reflectivity',
-        'reflectivity': 'uncorrected_reflectivity_h',  # need to change to input_reflectivity
+        'input_zdr': 'attenuation_corrected_differential_reflectivity',
+        'reflectivity': 'attenuation_corrected_reflectivity_h',  # need to change to input_reflectivity
```
Comment (scollis) [inline cmac/default_config.py]: nope.. should not be the attenuation corrected

Comment (scollis) [reply cmac/default_config.py]: Is there just a corrected_differential_reflectivity?

File: cmac/default_config.py [hunk 2]
```diff
@@ -550,19 +550,17 @@
...
         'd_coef': 1.804,
         'beta_coef': 0.64884,  # ZDR corrections
         'flip_phidp': True,
         'phidp_flipped': ['uncorrected_differential_phase','differential_phase'],
-        'zdr_offset': -3.8,
-        'offset_zdrs': ['differential_reflectivity_lag_1', 'differential_reflectivity'],
         'mbfs': cacti_csapr2_ppi_mbfs,
         'hard_const': cacti_csapr2_ppi_hard_const,
         'gen_clutter_from_refl': True,
+        'ref_offset': 0.0,
```
Comment (scollis) [inline cmac/default_config.py:563]: Yes

**Review by scollis — CHANGES_REQUESTED** [review]

**Comment by zssherman** [comment]
@scollis These are the fields in the level b file:
'attenuation_corrected_differential_reflectivity', 'attenuation_corrected_differential_reflectivity_lag_1', 'attenuation_corrected_reflectivity_h', 'censor_mask', 'classification_mask', 'copol_correlation_coeff', 'differential_phase', 'differential_reflectivity', 'differential_reflectivity_lag_1', 'mean_doppler_velocity', 'mean_doppler_velocity_v', 'normalized_coherent_power', 'normalized_coherent_power_v', 'reflectivity', 'reflectivity_v', 'signal_to_noise_ratio_copolar_h', 'signal_to_noise_ratio_copolar_v', 'specific_attenuation', 'specific_differential_attenuation', 'specific_differential_phase', 'spectral_width', 'spectral_width_v', 'uncorrected_copol_correlation_coeff', 'uncorrected_differential_phase', 'uncorrected_differential_reflectivity', 'uncorrected_differential_reflectivity_lag_1', 'uncorrected_mean_doppler_velocity_h', 'uncorrected_mean_doppler_velocity_v', 'uncorrected_reflectivity_h', 'uncorrected_reflectivity_v', 'uncorrected_spectral_width_h', 'uncorrected_spectral_width_v', 'unthresholded_power_copolar_h', 'unthresholded_power_copolar_v'

File: cmac/default_config.py (hunk 1 above)
Comment (zssherman) [reply cmac/default_config.py]: @scollis Not that I see, i posted the level b fields below

**Comment by zssherman** [comment]
Oh wait @scollis it should just be differential_reflectivity and reflectivity, the uncorrected versions are there and the corrected just aren't call corrected, correct?

**Comment by scollis** [comment]
Thanks.. hang 10

**Comment by scollis** [comment]
Yes.. just ref etc... He specifically says uncorrected for the raw.

**Comment by scollis** [comment]
Damn I love GitHub review

**Comment by zssherman** [comment]
I don't know why I thought it was the attenuated, if the code we are using is the attenuation correction...

File: cmac/cmac_radar.py [hunk 3]
```diff
@@ -157,14 +156,26 @@ def cmac(radar, sonde, config, geotiff=None, flip_velocity=False,
...
+        gate_data = radar.fields['gate_id']['data'].copy()
+        radar.fields['gate_id']['data'][clutter_data == 8] = 5
+        radar.fields['gate_id']['data'][clutter_data == 16] = 5
+        radar.fields['gate_id']['data'][clutter_data == 4] = 0
+        radar.fields['gate_id']['data'][clutter_data == 1] = 0
+        radar.fields['gate_id']['data'][clutter_data == 2] = 0
+        radar.fields['gate_id']['data'][gate_data == 0] = 0
+        notes = radar.fields['gate_id']['notes']
+        radar.fields['gate_id']['notes'] = notes + ',5:clutter'
+        radar.fields['gate_id']['valid_max'] = 5
```
Comment (scollis) [inline cmac/cmac_radar.py:178]: Going to have to trust you on this

**Review by scollis — APPROVED** [review]

File: cmac/cmac_radar.py (hunk 3 above)
Comment (zssherman) [reply cmac/cmac_radar.py:178]: @scollis 
So the classification mask has flags as such:
 'flag_masks': array([ 1,  2,  4,  8, 16], dtype=int32),
 'flag_meanings': 'second_trip third_trip interference clutter sunspoke'
I was going to revisit this and ask so glad you reminded me, but I added 1, 2, 4 to multi trip and clutter to 5

Comment (zssherman) [reply cmac/cmac_radar.py:178]: class_mask

Comment (zssherman) [reply cmac/cmac_radar.py:178]: Interference is that also multi trip? It looks like multi trip in the scan:
cmac_four_panel_plot cacticsaprcmacppi c1 20181111 010003

File: cmac/default_config.py (hunk 2 above)
Comment (zssherman) [reply cmac/default_config.py:563]: Deleted your comment on accident, discussed with Bobby will do a change.

**Pull request merged**